### PR TITLE
Fix/fetch shared files

### PR DESCRIPTION
**Pull request by nettashviro**
fix the fetching of getSharedFIles. related to pr #204

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - add popup for convert docs.
 
+### Fixed
+- fix fetch shared files bug.
```
Comment (Shahar-Y) [inline CHANGELOG.md:15]:
```suggestion
- hotfix: ([49](https://github.com/meateam/drive-ui/pull/49/)) : fix fetch shared files bug.
```

**Review by Shahar-Y — CHANGES_REQUESTED** [review]
Just commit the change and it's approved

File: CHANGELOG.md (hunk 1 above)
Comment (nettashviro) [reply CHANGELOG.md:15]: done

**Comment by nettashviro** [comment]
> Just commit the change and it's approved

changed Changelog file, please approve the pr :)

**Review by Shahar-Y — APPROVED** [review]

**Review by Shahar-Y — CHANGES_REQUESTED** [review]
Did you test to see what happens when some of the files do fail? 
Is there no different error handling for that case? And if all fail, what error is presented?

**Review by yoavrozner — APPROVED** [review]
Good job!

**Comment by nettashviro** [comment]
> Did you test to see what happens when some of the files do fail?
> Is there no different error handling for that case? And if all fail, what error is presented?

I checked all the end cases, when one file fails a corresponding error returns and it returns within the array of failed files.

File: CHANGELOG.md [hunk 2]
```diff
@@ -90,10 +90,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - add popup for convert docs.
-
 - add pagination to shared files and external transferd.
 -
 
+### Fixed
+- hotfix: ([49](https://github.com/meateam/drive-ui/pull/49/)) : fix fetch shared files bug.
+
```
Comment (Shahar-Y) [inline CHANGELOG.md:98]: um... unreleased?

**Review by Shahar-Y — CHANGES_REQUESTED** [review]
I see you have conflicts in files.js

File: CHANGELOG.md (hunk 2 above)
Comment (nettashviro) [reply CHANGELOG.md:98]: that's what happend when you work with this shitty changelog lol

Comment (Shahar-Y) [reply CHANGELOG.md]: All shall be done by the CHANGELOG policy!

**Review by Shahar-Y — APPROVED** [review]

**Pull request merged**